pages/common/ocrmypdf.md

```diff
@@ -0,0 +1,28 @@
+# ocrmypdf
+
+> Generate a searchable PDF or PDF/A from a scanned PDF or an image of text.
+> More information: <https://ocrmypdf.readthedocs.io/en/latest/introduction.html>.
```

Comment:
```suggestion
> More information: <https://github.com/jbarlow83/OCRmyPDF>.
```

Let others discuss this one.

Reply:
Disagree. The help information provided at the repository is minimal. However, I will replace the introduction link with the cookbook link that is closer to the purpose of tldr.

Reply:
let others see before doing :)